File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.*;

class Main {
    static int n, k;
    
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in); 
        
        n = sc.nextInt();
        k = sc.nextInt();
        
        for (int i = 0; i < n; i++) {
            int now = sc.nextInt();
            if (now < k) {
                System.out.print(now + " ");
            }
        }
    }
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.kt
import java.util.*;

fun main(args: Array<String>) = with(Scanner(System.`in`)) {
    val n = nextInt()
    val x = nextInt()
    for (int in 1..n) {
        val now = nextInt()
        if (now < x) {
            println(now)
        }
    }
}